Agreements/External/DUBN_Ant.txt

```diff
@@ -1,9 +1,9 @@
-FIR_COPX:EIDW:*:*:*:*:DUBN:Antrim:23000:*:EIDW
+FIR_COPX:EIDW:*:*:*:*:DUBN:Antrim:23000:*:^EIDW
 
 FIR_COPX:*:*:NEVRI:EGAA:*:DUBN:Antrim:*:14000:NEVRI
 FIR_COPX:*:*:NEVRI:EGAC:*:DUBN:Antrim:*:14000:NEVRI
 FIR_COPX:*:*:NEVRI:EGAD:*:DUBN:Antrim:*:14000:NEVRI
 
-FIR_COPX:*:*:BOYNE:EGNS:*:DUBN:Antrim:*:13000:BOYNE
```

Comment:
this is not a climbing agreement - but a descending agreement?

Reply:
Should be descending (though the Agreed Levels Diagram needs the arrow - it doesn't show a level by point). And the EIDW outbounds should be climbing.

Fixed!